### ISSUE TITLE
Thumbnail Image not working on wms Layer

### ISSUE THREAD

**Issue by kuntalparbat** [consbio/Leaflet.Basemaps#11]
thumbnail is working fine to show tile layers but when I trying to open WMS layer it’s not showing Image thumbnail. 

Please see the screenshot: https://goo.gl/photos/HZWSQhTKLvWrvoaLA

```
L.tileLayer.wms('http://demo.opengeo.org/geoserver/ows?', {
    layers: 'ne:ne',
    label: 'WMS Map'
})
```

**Comment by brendan-ward** [comment]
@kuntalparbat thank you for reporting this.  

The issue is that the plugin expects that the `tileLayer` instance is a standard tile layer URL, into which it can paste in values for tile z, x, and y values.  The WMS `tileLayer` does not use the same URL scheme, and so the plugin ends up requesting an invalid URL.

I will need to do some research to see how we might approach this.  I think it should be straightforward for the plugin to detect that a WMS `tileLayer` is being used, and use alternative methods of constructing the URL to include image bounds instead of tile z, x, and y values.

**Comment by brendan-ward** [comment]
resolved by 8e0b78fc834d7ffbe62e4b129c52a1ccf52f8e54

very sorry for the delay fixing this @kuntalparbat 

The geoserver demo URL you provided is not working today, so I found an alternative WMS, and added that to the readme example and index.html.  

Hope this works for you!
